apps/docs/content/guides/getting-started/quickstarts/hono.mdx

```diff
@@ -8,12 +8,6 @@ hideToc: true
 <StepHikeCompact>
 
   <StepHikeCompact.Step step={1}>
-    
-    <QuickstartDbSetup />
-
-  </StepHikeCompact.Step>
-
-  <StepHikeCompact.Step step={2}>
```

Comment:
Might need to update the step number on 3, 4, and 5 to 2, 3, and 4.

Reply:
Ahh, good catch, thank you!